samples/.NET/NUnit/README.md

```diff
@@ -33,9 +33,8 @@ If you have not yet created a workspace, please follow the [Get Started guide](.
 
 1. Set access token generated above as environment variable for your project: 
     ```powershell
-    $env:PLAYWRIGHT_SERVICE_ACCESS_KEY= # Paste Access Token value from previous step
+    $env:PLAYWRIGHT_SERVICE_ACCESS_TOKEN= # Paste Access Token value from previous step
     ```
```

Comment:
should we still add comment saying version below 1.40 would need to use PLAYWRIGHT_SERVICE_ACCESS_KEY